src/components/InventoryManager/InventoryManagerScreen.js

```diff
@@ -136,17 +266,34 @@ export default function InventoryManagerScreen() {
                 <DeleteButton deleteRecords={deleteSelectedRecords} />
```

Comment:
The delete listing button kind of ends in an awkward place, could you edit the styling to move it to the left so the button endsapproximately where the background rectangle (surrounding the listing cards) underneath it ends?